experiments/AtmosLES/bomex_model.jl

```diff
@@ -475,7 +493,8 @@ function bomex_model(
         param_set;
         problem = problem,
         turbulence = SmagorinskyLilly{FT}(C_smag),
-        moisture = EquilMoist{FT}(; maxiter = 5, tolerance = FT(0.1)),
+        #moisture = EquilMoist{FT}(; maxiter = 5, tolerance = FT(0.1)),
+        moisture = NonEquilMoist(),
```

Comment:
The default bomex model will now use non-equilibrium thermodynamics? 😅 @kpamnany, @yairchn should we expose this as a kwarg to `bomex_model`? The EDMF model has only been tested on the `EquilMoist` so far.

Reply:
Haha - no, sorry. This is me still testing. But it would be good to be able to switch between the two setups. I'll add that.

Reply:
@charleskawczynski - I added the optional `--moisture-model` keyword argument to bomex.